vendor/github.com/russross/blackfriday/LICENSE.txt

```diff
@@ -1,29 +0,0 @@
-Blackfriday is distributed under the Simplified BSD License:
```

Comment:
Ok, this PR should be correct now.

I'm sending a separate PR with the fixes to the `Makefile`.